Source/relay/TourGuide/Quests/Level 11 - Desert.ash

```diff
@@ -175,6 +175,15 @@ void QLevel11DesertGenerateTasks(ChecklistEntry [int] task_entries, ChecklistEnt
         else
             subentry.entries.listAppend("Use your desert sightseeing pamphlets. (+15% exploration)");
     }
+    int campgroundMilestoneCount = __campground[$item[milestone]];
+    if (campgroundMilestoneCount > 0) {
+        string milestonesPlural = campgroundMilestoneCount == 1 ? "" : "s";
+        subentry.entries.listAppend(HTMLGenerateSpanFont("Harvest your rock garden milestone" + milestonesPlural + "!", "red"));
+    }
+    if ($item[milestone].available_amount() > 0 && $item[milestone].item_is_usable()) {
+        string milestonesPlural = $item[milestone].available_amount() == 1 ? "" : "s";
+        subentry.entries.listAppend("Use your milestone" + mileStonesPlural + ". (+5% exploration each)");
```

Comment:
should we add anything here regarding why users should consider using the milestones early rather than late, for page accumulation reasons? i guess it's not strictly necessary but it feels like it might go here. similarly we could add a "Maybe wait until day 2 so you can get 10% advancement from Milestones and skip directly to worm-riding manual page accumulation" note, similar to the shen snakes d2 vs d1 note (which, it is worth noting, is very out of date and wrong. but it would go OK here!